examples/gadgets/grpc/custom_operator/main.go

```diff
@@ -68,11 +68,7 @@ func do() error {
 	}
 	defer runtime.Close()
 
-	params := map[string]string{
-		// Capture events coming from the host too
-		"operator.LocalManager.host": "true",
-	}
-	if err := runtime.RunGadget(gadgetCtx, nil, params); err != nil {
+	if err := runtime.RunGadget(gadgetCtx, nil, nil); err != nil {
```

Comment:
localmanager isn't used in this example, so I thought to remove params to avoid confusion